be/src/exprs/cast_expr.cpp

```diff
@@ -1599,10 +1606,18 @@ Expr* VectorizedCastExprFactory::create_primitive_cast(ObjectPool* pool, const T
                 return nullptr;
             }
         }
-    } else if (is_binary_type(from_type) || is_binary_type(to_type)) {
-        LOG(WARNING) << "vectorized engine not support from type: " << type_to_string(from_type)
-                     << ", to type: " << type_to_string(to_type);
-        return nullptr;
+    } else if (is_binary_type(to_type)) {
+        if (is_string_type(from_type)) {
+            if (allow_throw_exception) {
+                return new VectorizedCastExpr<TYPE_VARCHAR, TYPE_VARBINARY, true>(node);
+            } else {
+                return new VectorizedCastExpr<TYPE_VARCHAR, TYPE_VARBINARY, false>(node);
+            }
+        } else {
+            LOG(WARNING) << "vectorized engine not support from type: " << type_to_string(from_type)
+                         << ", to type: " << type_to_string(to_type);
```

Comment:
I don't think `vectorized engine` is still necessary because there is no other engine at all.
`Not support casting `

Reply:
Fixed